extension.js

```diff
@@ -53,6 +53,20 @@ const activate = async (context) => {
 		}, 3000)
 	})
 
+	//THIS IS WHERE STOP WATCH FUNCTION BEGINS
```

Comment:
A reminder that line 56 - 67 needs to be on a new branch with a corresponding ticket for each ie: "feature/#-commands" or "feature/#-resetTime", etc. however the teams decide to break up those tickets. ;)